Cargo.toml

```diff
@@ -101,7 +101,7 @@ fuel-tx = "0.58"
 fuel-vm = "0.58"
 
 # Dependencies from the `forc-wallet` repository:
-forc-wallet = "0.11"
+forc-wallet = "0.11.1"
 
```

Comment:
We're currently using only the first two numbers in SemVer.

Reply:
This is a common problem we have right now with only using "X.Y" versions. But here is the solution:

use `cargo update -p forc-wallet` :)

More explanation of the problem for more context:

## The Problem

Once pinned to "x.y" there is no way to incorporate minor bumps without running `cargo update` or first pinning to "x.y.z" and building the workspace and reverting back to "x.y". This is because `cargo` is not checking for a minor versions at each `cargo build` operation. 

This wouldn't be a problem if we were running cargo update in each release, but we are only running cargo update on each major bump of sway repo releases.


## Solution

So we need a way to only update specific dependencies, luckily cargo-update has a `-p` parameter to use with this. You can specifically only bump a single dependency if used like:

`cargo update -p forc-wallet`


## A way to improve

I think we can write a script like update_fuel_dependencies.sh which will essentially run `cargo update -p <fuel_dep_name>`, this can further be added to a git hook. Since we all adhere to semver no minor bump should break any code 🤞